package.json

```
@@ -232,4 +232,4 @@
     "last 2 versions",
     "ie 11"
   ]
-}
+}
```

Can you add this line break back?
```suggestion
}

```